### ISSUE TITLE
Clone deployment manifest iiotedge is not permitted after deploying 2.8.3 Azure IIoT

### ISSUE THREAD

**Issue by aleducq** [Azure/iotedge#6522]
In order to add proxy settings to the edgeHub and edgeAgent, I had to clone the generated manifest iiotedge.

I have this message:

![image](https://user-images.githubusercontent.com/77726779/179717680-dff97b25-94d1-453b-a199-622e7d183fa3.png)

## Steps to Reproduce
1. click Clone button on iiotedge deployment manifest

As a workaround, I have created a new deployment from scratch but potentially I forgot some parameters and have errors due to this bad configuration

**Comment by aleducq** [comment]
I think that the targetedCount metrics is not well formatted (something had to change in Azure) and that is why the cloning is not enable.
When I try to recreate an identical deployment, I have this error:
![image](https://user-images.githubusercontent.com/77726779/179752681-cf0a8bbd-cd43-42e0-a20f-f911ca749029.png)

**Comment by yophilav** [comment]
Hey @aleducq ,
Sounds like you have it figured out?
If not, could you provide more details please?

**Comment by aleducq** [comment]
No I didn't figure it out. the comment I've made was something else (I've deleted it)
I still can't clone the deployment manifest.
What kind of details do you need ?

**Comment by yophilav** [comment]
@aleducq ,
Since we are going to request your problematic deployments manifest which contains sensitive data, please close this GitHub Issue and open a support ticket with Azure so we can safely exchange the information there. Thank you!

**Comment by aleducq** [comment]
I already have one TrackingID#2207010050001600
Last message from data engineer:

Hello Alexis,

I got this feedback from the IIoT Product Group engineer who was able to reproduce your issue.

I did not find a reasonable fix/workaround. Customer should open a GitHub issue: [Issues · Azure/Industrial-IoT (github.com)](https://urldefense.com/v3/__https:/github.com/Azure/Industrial-IoT/issues__;!!K62b8Q!L2yJYiq_zRxKy5srqgsC9_hkryMGamawZebBpfYnuPmJRvOrK6S79CBAGQOKDFJYP9jgScVDre2DtfS5cmDA7jhKshje-uM$) so we can continue the discussion there.
-------------------------------------------

I’m really sorry that MS cannot help you to find an explanation/solution during this support ticket. There’re much more IIoT experts on the GitHub forum mentioned above as I can reach MS-internally, therefore it would be the best option to post your issue there to get in contact with additional expert. Again, sorry that I cannot provide a better approach…

Best Regards
